maven/spec/dependabot/maven/version_spec.rb

```diff
@@ -356,7 +339,9 @@
           let(:version) { described_class.new("1-sp-1") }
           let(:other_version) { described_class.new("1-ga-1") }
 
-          it { is_expected.to eq(-1) }
+          it "returns 1", skip: "await clarification on trimming vs named qualifier ordering" do
```

Comment:
I find this a contradiction in the maven spec. In one place it says trim null like `final, ga, 0` and in another it says `"1-sp-1" > "1-ga-1"` . I've found that making this test spec breaks others. I'll check this with the maven version checking tool